docs/plugins/ingest-attachment.asciidoc

```diff
@@ -238,3 +238,58 @@ properties on this top level field will contain the value of the
 first attachment only. However, by specifying the
 `target_field` on to a value on `_ingest._value` it will correctly
 associate the properties with the correct attachment.
+
+[[ingest-attachment-raw-metadata]]
+==== Extracting all raw metadata from documents
+
+By default, the `ingest-attachment` plugin only extracts a subset of the most common metadata.
+
+If you want to get back all the raw metadata, you can set `properties` to `raw_metadata`.
+It will populate a subfield `raw_metadata` with all key/value pairs found as metadata in the document.
```

Comment:
About the number of fields generated, I agree. That's why I never added this feature to the mapper attachments plugin.
Here, we know that `ingest` can help to filter out some fields later on.

We can indeed add another property and instead of modifying `properties`, just have a `raw_properties` list which is by default `[ "_none_" ]`, but can be `[ "_all_" ]` or a list of specific properties to be included.

So users would write:

```json
{
    "attachment" : {
        "field" : "data",
        "raw_properties": [ "_all_" ]
    }
}
```

WDYT?